### DISCUSSION TITLE
How to prioritize a server?

### DISCUSSION THREAD

**Discussion by mowanotm** [DNSCrypt/dnscrypt-proxy#2660] [Q&A] [answered]
Suppose including multiple servers in "server_names=",  
does dnscrypt-proxy always picks the "Server with the lowest initial latency" to query first?  
How can I designate a server as primary and others as fallback?

**Comment by lifenjoiner** [comment, marked as answer]
What you expect is not designed at all. That's some other app's logic.
The latency ranking is updated on the fly according the recent queries it has finished. Read the comments about `lb_strategy` and `lb_estimator`.
